### DISCUSSION TITLE
NativePHP native:serve not working

### DISCUSSION THREAD

**Discussion by mani-damirchilou** [NativePHP/laravel#393] [Q&A] [answered]
hi i've installed nativephp on an existing project using laravel sail and when i try to start the dev server i get an error saying :

`/var/www/html/vendor/nativephp/electron/resources/js/node_modules/electron/dist/electron: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory`

every single time in every single project

**Comment by simonhamp** [comment]
This should probably be a [bug report](https://github.com/nativephp/laravel/issues/new?assignees=&labels=bug&projects=&template=bug.yml)

**Reply by mani-damirchilou** [reply]
I wanted to report it as a bug but it seemed more like a problem with my computer

**Reply by simonhamp** [reply]
Still worth reporting as a bug as others may experience it and the extra detail may help them find the issue and what the solution was

**Comment by simonhamp** [comment, marked as answer]
My guess is that the Sail container is missing some dependency that Electron depends on and this isn't a NativePHP issue.

My suggestion would be to not run NativePHP inside your container. It's going to be tricky for lots of reasons not least of which that your container is a virtualised and often headless environment, whereas NativePHP is about building and showing GUI apps.
